1_OCaml/TP/TP1.ipynb

```diff
@@ -18,16 +18,61 @@
     "Devinez le type des fonctions suivantes et vérifier avec OCaml :\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    let temp a = if a+1=1 then () else () in 
</pre><p>Pourquoi <code>if a+1=1</code>  au lieu de <code>if a = 0</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>